### PR TITLE
details of staging

### PR DESCRIPTION
**Pull request by sammachin**
Some of the details on the staging environment, mostly just covers how it differs from prod.

### CODE REVIEW

File: development/staging.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+It uses a separate AWS account ending in ..9937
+
+Ben or ZJ can provision a user account for this account.
+
+The services are running in EU-West-1.
+
+## Nodes
+The staging environment uses one node running on a t2.micro for the management app and a pair of t2.micro nodes for the projects cluster.
+
+t2.micro is the smallest instance that can be used with EKS.
```
Comment (hardillb) [inline development/staging.md:18]: These are now t2.small as t2.micro didn't have enough IP addresses

File: development/staging.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## AWS Account
+
+It uses a separate AWS account ending in ..9937
+
+Ben or ZJ can provision a user account for this account.
+
+The services are running in EU-West-1.
+
+## Nodes
+The staging environment uses one node running on a t2.micro for the management app and a pair of t2.micro nodes for the projects cluster.
```
Comment (hardillb) [inline development/staging.md:16]: t2.small

File: development/staging.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Email
+Amazon SES is setup on staging however it is still running in sandbox mode which means only verified address & domains can RECEIVE emails from it, this is currently limited to flowforge.com email addresses.
+
+There is no intention to move this from sandbox as this helps to limit access to staging.
+
+If you need to use another email address with staging then you should verify the address through SES in the AWS Console.
+
+## Forge App Login
+There is an admin user in the Developer vault of 1Password
+
```
Comment (hardillb) [inline development/staging.md:29]: Include comment about admin interface on `forge.flowforge.dev`

Comment (sammachin) [reply development/staging.md:29]: The url is in the 1Password record, didn't want to put a link to it in the public handbook

Comment (hardillb) [reply development/staging.md:29]: Doh, will delete comment

File: development/staging.md (hunk 2 above)
Comment (sammachin) [reply development/staging.md]: doh I thought we went nano>micro, corrected

**Review by hardillb — APPROVED** [review]

**Pull request merged**